src/hooks/useBsv.ts

```diff
@@ -97,6 +97,18 @@ export type Web3SignMessageRequest = {
   tag?: DerivationTag;
 };
 
+export type Web3EncryptRequest = {
+  message: string;
+  encoding: 'utf8' | 'hex' | 'base64';
+  hexPubKeys: string[];
+  tag?: DerivationTag;
+};
+
+export type Web3DecryptRequest = {
+  hexMessages: string[];
```

Comment:
Similar `hex` comment here, but I also wonder if this should be base64 instead, as Jad just changed in the getSignatures end-point. base64 is 33% smaller than hex, and if these messages can be large, that is something to think about.